BFF/src/main/kotlin/br/com/example/simple/screen/OnboardingScreen.kt

```diff
@@ -1,20 +1,78 @@
-package br.com.example.simple.screen
+package br.com.example.simple.builder
 
+import br.com.example.simple.constants.BLUE
+import br.com.example.simple.constants.LIGHT_GREY
+import br.com.example.simple.constants.WHITE
+import br.com.zup.beagle.core.Style
 import br.com.zup.beagle.ext.applyFlex
-import br.com.zup.beagle.widget.core.AlignSelf
+import br.com.zup.beagle.ext.applyStyle
+import br.com.zup.beagle.widget.action.SetContext
+import br.com.zup.beagle.widget.context.ContextData
+import br.com.zup.beagle.widget.context.expressionOf
 import br.com.zup.beagle.widget.core.Flex
-import br.com.zup.beagle.widget.core.JustifyContent
-import br.com.zup.beagle.widget.layout.Container
-import br.com.zup.beagle.widget.layout.Screen
-import br.com.zup.beagle.widget.layout.ScreenBuilder
+import br.com.zup.beagle.widget.layout.*
+import br.com.zup.beagle.widget.pager.PageIndicator
 import br.com.zup.beagle.widget.ui.Text
 
 class OnboardingScreen : ScreenBuilder {
-    override fun build() = Screen(
+    override fun build(): Screen {
+        return Screen(
                 child = Container(
                         children = listOf(
-                                Text("Hello Beagle").applyFlex(flex = Flex(alignSelf = AlignSelf.CENTER))
+                                PageView(
+                                        pageIndicator = PageIndicator(
+                                                numberOfPages = 3,
+                                                selectedColor = BLUE,
+                                                unselectedColor = LIGHT_GREY,
+                                                currentPage = expressionOf("@{numberPage}")
+                                        ),
+                                        children = listOf(
+                                                page1,
+                                                page2,
+                                                page3
+                                        ),
+                                        onPageChange = listOf(SetContext("numberPage", "@{onPageChange}")),
+                                        currentPage = expressionOf("@{numberPage}")
+                                ),
+                                PageIndicator(
+                                        numberOfPages = 3,
+                                        selectedColor = BLUE,
+                                        unselectedColor = LIGHT_GREY,
+                                        currentPage = expressionOf("@{numberPage}")
+                                )
+                        ),
+                        context = ContextData("numberPage", 0)
+                ).applyStyle(style = Style(
```

Comment:
esse style = é desnecessário
```suggestion
                ).applyStyle(Style(
```